gauntlet/packages/gauntlet-solana-contracts/src/core/proto/ocr2Proto.ts

```diff
@@ -0,0 +1,45 @@
+// Autogenerated from ocr2Proto.proto with Protobuf.toJSON()
```

Comment:
Let's rename the file as `ocr2.proto.ts` and move it from `./core` as these are contracts/ocr2 specific, not part of core lib.